riscv_isac/plugins/specification.py

```diff
@@ -5,7 +5,7 @@
 
 class DecoderSpec(object):
     @decoderHookSpec
-    def setup(self,arch):
+    def setup(self,arch,isa):

```

Comment:
The setup function of the plugins does not need the `isa` anymore and all of the changes related need to be reverted back.

Reply:
Removed isa from the plugin setup function